pages/knowledge/colours-and-contrast/text/README.md

```diff
@@ -0,0 +1,79 @@
+---
+navigation_title: "Colour contrast for text"
+position: 3
+changed: "2010-04-21"
+---
+
+# Colour contrast for text
+
+**There is a general minimal colour contrast level for all kinds of text. As an exception, large text can have slightly lower contrast. This requirement applies to both “real” text and rasterised text on images.**
+
+## Minimal contrast ratio
+
+The [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#contrast-minimum) of `4.5:1` against adjacent colour(s) for text in general. An exception is made for large-scale text: It is considered easier to read and can therefore be set in a lower contrast ratio of at least `3:1`.
+
+Large-scale text is defined as:
+
+- Either **bold** text of at least 14 points (or 18.5 pixels).
```

Comment:
I'd love to keep the code examples like this:

> - For example `font-size: 14pt; font-weight: bold` (or `font-size: 18.5px; font-weight: bold`).

And

> - For example `font-size: 18pt` (or `font-size: 24px`).

It may feel obvious to professionals, but as accessibility even for long-term web designers often is a new and confusing topic at first, we still we feel that bridging the "gap" between the two worlds (accessibility and CSS) is very useful. ("Nützt's nüt, de schadt's nüt.")

Reply:
**To Do:** Add this.

Reply:
@jmuheim I decided to not change it after all. I think especially considering non-coders the CSS version would make things harder to understand
* It makes it really awkward to explain that these are minimum values. “At least `font-size: 14pt` or `font-size: 18.5px`” is maybe understandable but also not the most inviting sentence.
* I don’t want this to look like a code example ready for copy-pasting because people *shouldn’t* define their font-sizes in absolute values. Explaining the pros and cons of relative vs. absolute units goes beyond the scope of this topic. Writing it in text at least creates a certain layer of abstraction.